rpcs3/Input/sdl_instance.cpp

```diff
@@ -69,7 +69,7 @@ bool sdl_instance::initialize()
 	set_hint(SDL_HINT_JOYSTICK_HIDAPI_PS3, "1");
 #endif
 
-	if (!SDL_Init(SDL_INIT_JOYSTICK | SDL_INIT_GAMEPAD))
+	if (!SDL_Init(SDL_INIT_JOYSTICK | SDL_INIT_GAMEPAD | SDL_INIT_HAPTIC))
```

Comment:
SDL_INIT_GAMEPAD comes with SDL_INIT_JOYSTICK.
I already removed this on another unmerged PR

```suggestion
	if (!SDL_Init(SDL_INIT_GAMEPAD | SDL_INIT_HAPTIC))
```